addon/Config.jsm

```diff
@@ -72,9 +72,6 @@ var config = {
   // required LOG key
   "log": {
     // Fatal: 70, Error: 60, Warn: 50, Info: 40, Config: 30, Debug: 20, Trace: 10, All: -1,
-    "bootstrap":  {
-      "level": "Debug",
```

Comment:
Required for what? I know this goes to Log.jsm in StudyUtils... Not clear to me though why this is here?

Reply:
(I already pulled this out, and switched over to proper logging.